frontend/src/models/user.ts

```diff
@@ -0,0 +1,7 @@
+export interface User {
+    id: number
+    name: string,
```

Comment:
field name is inconsistent with your backend model and loggedUser model. 
You have lots of small errors. Most of them has to be caught by you